scripts/src/main/resources/scripts/functions

```diff
@@ -729,14 +740,22 @@ function doReplaceExtractedFile() {
   fi
   if [ "${target}" = "${DEVON_IDE_HOME}/scripts" ] && doIsWindows
   then
-    (sleep 10;doBackup "${target}" "${3}";doRunCommand "mv '${1}' '${target}'") &
+    # Extension fo msi and exe files
+    if [ "${ext}" = ".msi" ] || [ "${ext}" != ".exe" ]
+    then
+      (sleep 10;doBackup "${target}" "${3}";doRunCommand "mv '${1}' '${target}'") &
+    fi
```

Comment:
I do not see how this relates to msi and exe.
We can have a separate chat about this crazy `(sleep 10;...)` code.
It is only a workaround for windows file locks when devonfw-ide updates itself in case you call `devon ide update scripts`.
IMHO you should revert this change here (line 732/743)

Reply:
I change this part, but we have to look again at it